### PR TITLE
ETH Transactions GQL

### PR DESCRIPTION
**Pull request by vyzaldysanchez**
Closes: https://app.shortcut.com/chainlinklabs/story/21722/gql-endpoints-for-transactions-discussion

### CODE REVIEW

**Comment by cl-homu** [comment]
:umbrella: The latest upstream changes (presumably 8e9ce9b3540faaf191f7523c123bd9c6d43b17a9) made this pull request unmergeable. Please [resolve the merge conflicts](https://rustc-dev-guide.rust-lang.org/git.html#rebasing-and-conflicts).

File: core/web/resolver/eth_transaction.go [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+	chain, err := loader.GetChainByID(ctx, string(r.EVMChainID()))
+	if err != nil {
+		return nil, err
+	}
+
+	return NewChain(*chain), nil
+}
+
+func (r *EthTransactionResolver) Attempts() []*EthTransactionAttemptResolver {
+	return NewEthTransactionsAttempts(r.tx.EthTxAttempts)
```
Comment (jkongie) [inline core/web/resolver/eth_transaction.go:93]: It would be better to dataload this to reduce database load

**Review by jkongie — CHANGES_REQUESTED** [review]

File: core/web/resolver/eth_transaction.go (hunk 1 above)
Comment (vyzaldysanchez) [reply core/web/resolver/eth_transaction.go]: implemented

**Comment by cl-homu** [comment]
:umbrella: The latest upstream changes (presumably 08781b5273d096219a140fa27fd5e8e738bb2ef5) made this pull request unmergeable. Please [resolve the merge conflicts](https://rustc-dev-guide.rust-lang.org/git.html#rebasing-and-conflicts).

**Review by jkongie — APPROVED** [review]

**Review by jkongie — APPROVED** [review]

**Pull request merged**